phase2/src/bin/beacon.rs

```diff
@@ -0,0 +1,94 @@
+extern crate rand;
+extern crate phase2;
+extern crate memmap;
+extern crate num_bigint;
+extern crate num_traits;
+extern crate blake2;
+extern crate byteorder;
+extern crate exitcode;
+extern crate itertools;
+extern crate crypto;
+extern crate hex;
+
+use itertools::Itertools;
+
+use std::fs::File;
+use std::fs::OpenOptions;
+
+
+fn main() {
+    let args: Vec<String> = std::env::args().collect();
+    if args.len() != 5 {
+        println!("Usage: \n<in_params.params> <in_beacon_hash> <in_num_iterations_exp> <out_params.params>");
+        std::process::exit(exitcode::USAGE);
+    }
+    let in_params_filename = &args[1];
+    let beacon_hash = &args[2];
+    let num_iterations_exp = &args[3].parse::<usize>().unwrap();
+    let out_params_filename = &args[4];
+
+    let disallow_points_at_infinity = false;
```

Comment:
@kobigurk Would you also like `disallow_points_at_infinity`/`should_filter_points_at_infinity` to be command line inputs?